packages/components/src/core/Tag/index.stories.tsx

```diff
@@ -102,102 +102,241 @@ export const Default = {
+    }
+
+    // loop through all PSEUDO_STATES + create headers for COLORS
+    function TagHoverableOption({ sdsStyle, color }) {
+      return (
+        <div style={displayContents}>
+          <h2 style={colorLabel}>
+            Color: <b>{color[0].length === 1 ? color : "custom"}</b>
+          </h2>
+          {PSEUDO_STATES.map((state) => {
+            return (
+              <TagColor
+                sdsStyle={sdsStyle}
+                color={color}
+                state={state}
+                key={state}
+              />
+            );
+          })}
+        </div>
+      );
+    }
+
+    // loop through all HOVERABLE_OPTIONS + create headers for PSEUDO_STATES
+    function TagColor({ sdsStyle, color, state }) {
+      return (
+        <div style={displayContents}>
+          <h3 style={stateLabel}>
+            Pseudo state: <b>{state}</b>
+          </h3>
+          {HOVERABLE_OPTIONS.map((hover, hoverIndex) => {
+            return (
+              <TagType
+                sdsStyle={sdsStyle}
+                color={color}
+                state={state}
+                hover={hover}
+                key={hover}
+                hoverIndex={hoverIndex}
+              />
+            );
+          })}
+        </div>
+      );
+    }
+
+    // loop through all SDS_TYPES + create headers for HOVERABLE_OPTIONS
+    function TagType({ sdsStyle, color, state, hover, hoverIndex }) {
+      return (
+        <div style={displayContents}>
+          <h6 style={hoverLabel}>
+            Hoverable: <b>{hover ? "true" : "false"}</b>
+          </h6>
+          {SDS_TYPES.map((type) => {
+            return (
+              <TagIconOptions
+                sdsStyle={sdsStyle}
+                color={color}
+                state={state}
+                hover={hover}
+                hoverIndex={hoverIndex}
+                type={type}
+                key={type}
+              />
+            );
+          })}
+        </div>
+      );
+    }
+
+    // loop through all ICON_OPTIONS + create headers for SDS_TYPES, ICON_OPTIONS
+    function TagIconOptions({
+      sdsStyle,
+      color,
+      state,
+      hover,
+      hoverIndex,
+      type,
+    }) {
+      return (
+        <div style={typeLevel}>
+          {hoverIndex === 0 && (
+            <h5 style={typeLabel}>
+              Type: <b>{type}</b>
+            </h5>
+          )}
+          {ICON_OPTIONS.map((icon, index) => {
+            let iconText = icon ? "yes" : "no";
+            return (
+              <div style={index % 2 ? { iconLevel1 } : { iconLevel2 }}>
+                {hoverIndex === 0 && (
+                  <h6 style={iconLabel}>
+                    Icon: <b>{iconText}</b>
+                  </h6>
+                )}
+                <RawTag
+                  {...props}
+                  data-testid="tag"
+                  sdsStyle={sdsStyle}
+                  color={color}
+                  className={hover ? `pseudo-${state}` : `pseudo-default`}
```

Comment:
Here's the issue I mentioned in the PR description about the `hover` prop being overridden by the pseudo state addon, so I temporarily overrode it myself such that the tags that have `hover={false}` would always render with the default pseudo state...